.circleci/integration-tests/master_dag.py

```diff
@@ -199,7 +199,20 @@ def prepare_dag_dependency(task_info, execution_time):
         ],
     )
 
-    airflow_version_check = (get_airflow_version, check_logs_data)
+    # The below task `check_non_ascii_data` is specifically added to test the astronomer logging
+    # providers for non-ASCII characters, as we have providers' tests running on all
+    # cloud providers, and we do not want multiple deployments for this.
+    check_non_ascii_data = PythonOperator(
+        task_id="check_non_ascii_data",
+        python_callable=check_log,
```

Comment:
Where is this callable check_log defined?

Reply:
we were already using this callable from [here](https://github.com/astronomer/astronomer-providers/blob/a49fe81d86ae49e4ddba7ae988914d703e79c05f/.circleci/integration-tests/airflow_dag_introspection.py#L36)

Reply:
sorry forgot to check the imports.